### PR TITLE
Update heroku docs in getting started

### PR DESCRIPTION
**Pull request by shaftoe**
I was not that familiar with Heroku add-ons so I had to go through some documentation, I hope this will help new Ackee users willing to deploy on Heroku

### CODE REVIEW

File: docs/Get started.md [hunk 1]
```diff
@@ -170,7 +170,9 @@ Simply deploy to Heroku by clicking this button:
 
 ### 2. Configure Ackee
 
-Ensure that you're using the correct CORS headers by setting [`ACKEE_ALLOW_ORIGIN`](CORS%20headers.md#heroku-or-platforms-as-a-service-configuration).
+- You need to have a MongoDB instance running, for example using [mLab MongoDB](https://devcenter.heroku.com/articles/mongolab) add-on which is currently available for free. This is as simple as typing `heroku addons:create mongolab --app <YOUR_APP_NAME>` using the CLI, or using the web dashboard; more details at the [official documentation](https://devcenter.heroku.com/articles/managing-add-ons). The add-on will inject `MONGODB_URI` variable into the Ackee application so you just need to restart Heroku app _dyno_ to make Ackee process connect to MongoDB. NOTE: you need to provide your credit card details in order to use add-ons in Heroku.
```
Comment (aleccool213) [inline docs/Get started.md:173]: You should suggest a different add-on provider as mLab is being deprecated soon.

Reference: https://devcenter.heroku.com/changelog-items/1823

Comment (shaftoe) [reply docs/Get started.md:173]: @aleccool213 can you recommend any?

Comment (aleccool213) [reply docs/Get started.md]: Not yet 😟  I have yet to switch off of mLab.

Comment (shaftoe) [reply docs/Get started.md]: @aleccool213 yeah, ObjectRocket offering is too expensive for most basic needs, I wonder if it's possible/makes sense to create an Heroku add-on to run it on a free tier dyno

**Comment by electerious** [comment]
@shaftoe Thanks for the PR and thanks @aleccool213 for approving the changes! 👍

**Comment by shaftoe** [comment]
@aleccool213 @electerious FYI, I'm testing the free [MongoDB Atlas](https://cloud.mongodb.com/) offer and it looks like a decent replacement for mLab's one for basic usage (512MB free storage, shared host), even though it doesn't come with any native Heroku integration.

HTH

**Comment by electerious** [comment]
@shaftoe Sounds good! Let us know how it performances and if 512mb are enough. Might be worth to reduce the data use in Ackee if not.

**Comment by shaftoe** [comment]
@electerious will do!
Also, I wonder if there's an easy way to prune old records from MongoDB, personally I'm only interested in having the last month or so available from the web interface. Probably some __DANGER ZONE__ section in the web client with a "remove older than X days" feature would be best.

**Pull request merged**